manifests/init.pp

```diff
@@ -66,7 +68,7 @@
   }
 
   if $pool == undef {
-    $command = "/usr/sbin/subscription-manager register --name=\"${::fqdn}\"  --username=\"${rh_user}\" --password=\"${rh_password}\" --auto-attach ${proxycli} && /usr/sbin/subscription-manager repo-override --repo rhel-${::operatingsystemmajrelease}-server-optional-rpms --add=enabled:1 && /usr/sbin/subscription-manager repo-override --repo rhel-${::operatingsystemmajrelease}-server-extras-rpms --add=enabled:1"
+    $command = "/usr/sbin/subscription-manager register --name=\"${::fqdn}\"  --username=\"${rh_user}\" --password=\"${rh_password}\" --auto-attach ${proxycli}"
```

Comment:
Well fqdn is a built in top scope variable, so that's safe. As for username and password, that might entering regex territory trying to check for whatever Red Hat requires as valid username and password. Checking those might be going over the top. We can always have single quotes just for these two. If you agree I'll commit that right away.